pages.pt-BR/common/7z.md

```diff
@@ -0,0 +1,32 @@
+# 7z
+
+> Um compactador de arquivos com alta taxa de compressão.
+> Página oficial: <https://www.7-zip.org/>.
+
+- Compactar um arquivo ou diretório:
+
+`7z a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretorio}}`
+
+- Criptografar um arquivo existente (incluindo cabeçalhos):
+
+`7z a {{arquivo_criptografado.7z}} -p{{senha}} -mhe {{arquivo.7z}}`
+
+- Descompactar um arquivo mantendo a estrutura de diretórios original:
+
+`7z x {{arquivo.7z}}`
+
+- Descompactar um arquivo em um diretório específicado pelo usuário:
+
+`7z x {{arquivo.7z}} -o{{caminho/diretorio}}`
```

Comment:
```suggestion
`7z x {{arquivo.7z}} -o {{caminho/diretorio}}`
```

Reply:
This change doesn't seem to have been applied, @andrik -- do you disagree? Does it not work?

Also, why not include the accent in "diretório"?